schedule/yast/xfs/xfs_spvm.yaml

```diff
@@ -25,7 +25,8 @@ schedule:
   - installation/clock_and_timezone/accept_timezone_configuration
   - installation/authentication/use_same_password_for_root
   - installation/authentication/default_user_simple_pwd
-  - installation/installation_overview
+  - installation/installation_settings/validate_ssh_service_enabled
+  - installation/installation_settings/open_ssh_port
```

Comment:
looks like we miss here for s390x: https://openqa.suse.de/tests/7748335#step/installation_overview/18

Reply:
The schedule for s390x is included to PR: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/4b6ba1d4633929a6be2a4486f74d9651d5d9b7e9/schedule/yast/xfs/xfs%40svirt-s390x.yaml